### ISSUE TITLE
Is there a way to have a hotkey be in series (aka cmd-k cmd-t)

### ISSUE THREAD

**Issue by tnrich** [palantir/blueprint#3392]
Just like the title says, I want a hotkey that is a series of multiple presses. 

Here's what that looks like in vscode for example: 

![image](https://user-images.githubusercontent.com/2730609/53773045-914c6000-3e9d-11e9-9cf3-79451d399195.png)

Can we do that in blueprint?

**Comment by adidahiya** [comment]
No, this is not possible right now, and there isn't a proposal on the table yet for adding this feature. I would consider a PR if this is useful to you.

**Comment by tnrich** [comment]
This isn't crucial for me but it seems like something that should be supported. Thanks for the info!
